docs/implementation.adoc

```diff
@@ -357,6 +357,18 @@ public class AccurevSCMSource extends SCMSource {
                     new NonLocalizable("Streams")
                     // Better: Messages._AccurevSCMSource_StreamHeadCategory()
                 ),
+                new UncategorizedSCMSourceCategory(
```

Comment:
you can't have more than one `UncategorizedSCMSourceCategory`

Reply:
A gated stream is exactly like a stream however what happens underneath is different. A gated stream spawns staging stream for each changing stream that is underneath and wants to promote changes. Could be a workspace or a regular stream or even another gated stream is what's